crowdsec-docs/unversioned/integrations/mikrotik.mdx

```diff
@@ -0,0 +1,122 @@
+---
+id: mikrotik
+title: Mikrotik
+---
+
+import ThemedImage from "@theme/ThemedImage";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+The CrowdSec Mikrotik integration allows you to block malicious IPs in your Mikrotik router. This guide will walk you through the steps to integrate CrowdSec blocklists with your Mikrotik router.
+
+### Prerequisites
+
+Before you begin, please ensure your Mikrotik device supports ingesting blocklists. If you are unsure, please refer to the Mikrotik documentation or contact Mikrotik support.
+
+### Steps
+
+We will presume you followed the [Getting Started](integrations/intro.mdx) guide and have created an account on the CrowdSec Console.
+
+Once you are authenticated, you can proceed to the Blocklist tab located on the top menu bar, from there you can select the Integrations sub menu.
+
+Once the page has loaded, you can click the "Connect" button under the Mikrotik logo.
+
+<ThemedImage
```

Comment:
Don't know if it still WIP but this image is not loading

Reply:
fixed